.github/renovate.json

```
@@ -1,8 +1,13 @@
 {
   "extends": [
-    "config:base",
-    ":preserveSemverRanges"
+    "config:base"
```

Should add a schedule, no ?
```suggestion
    "config:base",
    "schedule:earlyMondays"
```

---

True!

---

Done